assets/js/main.js

```diff
@@ -1,15 +1,19 @@
-$("body").on({
+$('body').on({
   mouseenter: function() {
     var submenu = $(this).find(".sub-menu");
 
+    if(!$(this).hasClass('submenu-item'))
+      $('.header-main').removeClass('header-open');
```

Comment:
Very minor, could you use always curly braces like in other places?

Reply:
Sure!